src/Common/src/System/Globalization/IdnMapping.Unix.cs

```diff
@@ -11,44 +11,37 @@ private unsafe string GetAsciiCore(char* unicode, int count)
             uint flags = Flags;
             CheckInvalidIdnCharacters(unicode, count, flags, nameof(unicode));
 
+            int length = Interop.GlobalizationNative.ToAscii(flags, unicode, count, null, 0);
```

Comment:
No we don't have measurements. this is the classical debate between the speed and the memory allocation. 

I believe on Windows we already calling it twice too and this is why I said let's keep calling twice as we didn't get complains before. 
it is expected to have GetAscii called with short strings in the real scenarios so extra allocations can work too.